pyiron_atomistics/lammps/base.py

```diff
@@ -983,7 +983,7 @@ def collect_dump_file(self, file_name="dump.out", cwd=None):
                 pd.read_csv(
                     StringIO("\n".join(dump[llst:llen]).replace("ITEM: ATOMS ", "")),
                     delim_whitespace=True,
-                )
+                ).sort_values(by='id')
```

Comment:
```suggestion
                ).sort_values(by="id", ignore_index=True)
```
This simply makes sure that the pandas index is ascending.  Otherwise there might be issues if someone later indexes the `DataFrame` naively.